ReactWindows/ReactNative.Net46/Tracing/NullLoggingActivityBuilder.cs

```diff
@@ -15,8 +15,13 @@ public IDisposable Start()
         {
             return new NullDisposable();
         }
+
+        public NullDisposable Create()
+        {
+            return new NullDisposable();
+        }
         
-        class NullDisposable : IDisposable
+        public class NullDisposable : IDisposable
```

Comment:
>public [](start = 8, length = 6)

It's usually not kosher to have public nested classes for C# (we made an exception to reuse the builder pattern for ReactInstanceManager and a few other things, but the only exception so far has been for the builder pattern)